extension/goplugin/schemas.go

```diff
@@ -458,25 +464,38 @@ func (schema *Schema) DbStateUpdateRaw(rawResource interface{}, requestContext g
 
 // DeleteRaw deletes resource by ID
 func (schema *Schema) DeleteRaw(id string, context goext.Context) error {
-	return schema.delete(goext.Filter{"id": id}, context, true)
+	return schema.delete(goext.Filter{"id": id}, context)
 }
 
 // DeleteFilterRaw deletes resource by filter
 func (schema *Schema) DeleteFilterRaw(filter goext.Filter, context goext.Context) error {
-	return schema.delete(filter, context, true)
+	return schema.delete(filter, context)
 }
 
 // DbDeleteRaw deletes resource by ID without triggering events
 func (schema *Schema) DbDeleteRaw(id string, context goext.Context) error {
-	return schema.delete(goext.Filter{"id": id}, context, false)
+	return schema.dbDeleteFilter(goext.Filter{"id": id}, context)
 }
 
 // DbDeleteFilterRaw deletes resource by filter without triggering events
 func (schema *Schema) DbDeleteFilterRaw(filter goext.Filter, context goext.Context) error {
-	return schema.delete(filter, context, false)
+	return schema.dbDeleteFilter(filter, context)
+}
+
+func (schema *Schema) dbDeleteFilter(filter goext.Filter, ctx goext.Context) error {
```

Comment:
Please rename `dbDeleteFilter` to `dbDelete` or rename `delete` to `deleteFilter`, because now these names are inconsistent.

Reply:
done